pkg/sql/distsql_running.go

```diff
@@ -50,18 +52,29 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/util/metric"
 	"github.com/cockroachdb/cockroach/pkg/util/mon"
 	"github.com/cockroachdb/cockroach/pkg/util/ring"
+	"github.com/cockroachdb/cockroach/pkg/util/stop"
 	"github.com/cockroachdb/cockroach/pkg/util/syncutil"
 	"github.com/cockroachdb/cockroach/pkg/util/tracing"
 	"github.com/cockroachdb/errors"
 	pbtypes "github.com/gogo/protobuf/types"
 )
 
-// To allow queries to send out flow RPCs in parallel, we use a pool of workers
-// that can issue the RPCs on behalf of the running code. The pool is shared by
-// multiple queries.
-const numRunners = 16
+var settingDistSQLRunnersCPUMultiple = settings.RegisterIntSetting(
+	settings.TenantWritable,
+	"sql.distsql.runners_cpu_multiple",
+	"the value multiplied by the number of CPUs on a node to determine "+
```

Comment:
do we ever imagine wanting to set this to less than numCores? 

you could have the value just be the desired worker per node count, and then have the default value be 4*numCores?